### PR TITLE
886555-UG documentation for the digital gauge control

### PR DESCRIPTION
**Pull request by SriRadheshNagS**
UG documentation for the digital gauge control.

Added the new feature digital gauge control

### CODE REVIEW

File: MAUI/DigitalGauge/migration.md [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+<td>Gets or sets the value for the character height</td>
+</tr>
+<tr>
+<td>CharacterSpacing</td>
+<td>CharacterSpacing</td>
+<td>Gets or sets the value for the character spacing</td>
+</tr>
+</table>
+
+The following code example explains how to initialize the axis in Xamarin SfDigitalGauge and .NET MAUI SfDigitalGauge.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/migration.md:129]: what axis?

File: MAUI/DigitalGauge/Customize-character-segments.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Customize disabled segment
+
+You can customize the color and opacity of disabled segments using the `DisabledSegmentStroke` and `DisabledSegmentStroke`properties. The width of the digital character value can be customized using the `StrokeWidth` property.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Customize-character-segments.md:169]: The width of the digital character value can be customized using the `StrokeWidth` property. - no need

File: MAUI/DigitalGauge/Customize-character-segments.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Customize character segment stroke
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Customize-character-segments.md:117]: Customize character segment stroke and stroke width

add it in here

The width of the digital character value can be customized using the StrokeWidth property.

File: MAUI/DigitalGauge/Character-Types.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Fourteen segment
+
+The fourteen-segment type is capable of displaying numbers and alphabet efficiently.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Character-Types.md:77]: The fourteen-segment type is capable of displaying numbers and the alphabet efficiently.

File: MAUI/DigitalGauge/Character-Types.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+## Sixteen segment
+
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Character-Types.md:80]: The sixteen-segment type is capable of displaying numbers and the alphabet clearly.

File: MAUI/DigitalGauge/Character-Types.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+The sixteen-segment type is capable of displaying numbers and alphabet clearly.
+
+## EightCrossEightDotMatrix segment
+
+The dot matrix segment type is capable of displaying numbers, alphabet, and special characters efficiently.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Character-Types.md:85]: The dot matrix segment type is capable of displaying numbers, the alphabet, and special characters efficiently.

File: MAUI/DigitalGauge/Customize-character-segments.md [hunk 7]
```diff
@@ -0,0 +1,268 @@
...
+documentation: ug
+---
+
+# Customize Character Segments in Maui Digital Gauge (SfDigitalGauge)
+
+The characters of a digital gauge can be customized in terms of `CharacterWidth`, `CharacterHeight`, `CharacterSpacing`, and `CharacterStrokeColor`.
+
+## customize character size
+
+The values of digital characters are scaled by altering the height and width of digital characters. It is achieved by setting the `CharacterHeight`and `CharacterWidth` properties in the digital gauge. Default value of `CharacterHeight` and `CharacterWidth` is 25.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Customize-character-segments.md:16]: The values of digital characters are scaled by altering the height and width of digital characters. This is achieved by setting the `CharacterHeight` and `CharacterWidth` properties in the digital gauge. The default value of `CharacterHeight` and `CharacterWidth` is 25.

Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Customize-character-segments.md:16]: default characterheight is 60
default characterwidth is 40

File: MAUI/DigitalGauge/Customize-character-segments.md [hunk 8]
```diff
@@ -0,0 +1,268 @@
...
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Setting character spacing
+
+The values of digital characters are spaced by altering the space of digital characters. It is achieved by setting the `CharacterSpacing` property.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Customize-character-segments.md:67]: The values of digital characters are spaced by altering the space between digital characters. This is achieved by setting the `CharacterSpacing` property.

File: MAUI/DigitalGauge/Display-Types.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+documentation: ug
+---
+
+# Display Types in Maui Digital Gauge (SfDigitalGauge)
+
+The digital gauge displays numbers, alphabet, and special characters, which are given in the text property.
+
+## Setting value to number
+
+Numbers can be displayed in digital gauge in different formats using the `CharacterType`property.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Display-Types.md:16]: Numbers can be displayed in a digital gauge in different formats using the `CharacterType` property.

File: MAUI/DigitalGauge/Display-Types.md [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Setting value to alphabet
+
+Alphabet can be displayed in digital gauge using any one of the character format types.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Display-Types.md:68]: Alphabet can be displayed in a digital gauge using any one of the character format types.

File: MAUI/DigitalGauge/Getting-Started.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title: Getting Started with Maui Digital Gauge control | Syncfusion
+description: Learn here about getting started with Syncfusion Maui Digital Gauge (SfDigitalGauge) control, its elements and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+# Getting Started with Maui Digital Gauge (SfDigitalGauge)
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Getting-Started.md:9]: FYR - https://help.syncfusion.com/maui/stepprogressbar/getting-started

File: MAUI/DigitalGauge/Getting-Started.md [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+You can add SfDigitalGauge reference using one of the following methods:
+
+**Method 1: Adding SfDigitalGauge reference from nuget.org**
+
+Syncfusion Maui components are available in [nuget.org](https://www.nuget.org/). To add SfDigitalGauge to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Gauges](https://www.nuget.org/packages/Syncfusion.Maui.Gauges/25.2.4?_src=template), and then install it.
+
+N> Install the same version of SfDigitalGauge NuGet in all the projects.
+
+
+## Adding namespace for the assemblies
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Getting-Started.md:24]: after register handler add this code

File: MAUI/DigitalGauge/migration.md [hunk 13]
```diff
@@ -0,0 +1,367 @@
...
+SfDigitalGauge digitalGauge = new SfDigitalGauge();
+this.Content = digitalGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+## Character Segments
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/migration.md:78]: ## Properties

The following code example, explains how to initialize the properties of the `Xamarin SfDigitalGauge` and `.NET MAUI SfDigitalGauge.`

File: MAUI/DigitalGauge/migration.md [hunk 14]
```diff
@@ -0,0 +1,367 @@
...
+
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+## Character Types
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/migration.md:234]: add these topic before Character Type

## Enum
https://help.syncfusion.com/cr/xamarin/Syncfusion.SfGauge.XForms.CharacterType.html

File: MAUI/DigitalGauge/migration.md [hunk 15]
```diff
@@ -0,0 +1,367 @@
...
+
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/migration.md:367]: after the add Topic

## Event

https://help.syncfusion.com/cr/xamarin/Syncfusion.SfGauge.XForms.SfDigitalGauge.html#Syncfusion_SfGauge_XForms_SfDigitalGauge_ValueChanged

File: MAUI/DigitalGauge/overview.md [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+layout: post
+title: About Maui Digital Gauge control | Syncfusion
+description: Learn here all about introduction of Syncfusion Maui Digital Gauge (SfDigitalGauge) control, its elements and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+# Maui Digital Gauge (SfDigitalGauge) Overview
+
+The digital gauge control is used to display alphanumeric characters in digital (LED display) mode. This control displays a range of values that uses character in combination with numbers.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/overview.md:11]: The Syncfusion .NET MAUI DigitalGauge control is used to display alphanumeric characters in digital (LED display) mode. This control displays a range of values that use characters in combination with numbers.

File: MAUI/DigitalGauge/overview.md [hunk 17]
```diff
@@ -0,0 +1,40 @@
...
+•	16-segment display
+•	8*8 dot matrix display
+
+### 7-segment display
+
+This type of digital gauge displays digital characters in 7 segments. It is mainly used to display numbers.
+
+### 14-segment display
+
+This type of digital gauge displays digital characters in 14 segments. It is mainly used to display both the numbers and alphabet.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/overview.md:28]: This type of digital gauge displays digital characters in 14 segments. It is mainly used to display both numbers and the alphabet.

File: MAUI/DigitalGauge/overview.md [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+
+### 16-segment display
+
+This type of digital gauge displays digital characters in 16 segments. It is also mainly used to display both the numbers and alphabet.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/overview.md:32]: This type of digital gauge displays digital characters in 16 segments. It is also mainly used to display both numbers and the alphabet.

File: MAUI/DigitalGauge/overview.md [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+
+### 8 * 8 dot matrix display
+
+This type of digital gauge displays digital characters in 8*8 dot matrix segments, in which the characters are spotted by the regular brush and the remaining dots are spotted by the dimmed brush. It is mainly used to display numbers, characters, and special characters.
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/overview.md:36]: This type of digital gauge displays digital characters in 8x8 dot matrix segments, where the characters are spotted by the regular brush and the remaining dots are spotted by the dimmed brush. It is mainly used to display numbers, characters, and special characters.

File: MAUI/DigitalGauge/Getting-Started.md [hunk 20]
```diff
@@ -0,0 +1,225 @@
...
+    }
+}
+
+{% endhighlight %} 
+
+## Setting character type for digital gauge
+
+By using the `CharacterType` property, you can set the segments for digital gauge. The digital characters can be drawn in the following four different segments:
+
+•	SevenSegment
```
Comment (saiganesh-sakthivel) [inline MAUI/DigitalGauge/Getting-Started.md:104]: remove the empty spacing between sevensegment, 14segment, 16 segment

File: MAUI/DigitalGauge/Character-Types.md [hunk 21]
```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Character Types in Maui Digital Gauge control | Syncfusion
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Character-Types.md:3]: Character Types in .NET MAUI Digital Gauge control

**Use** .NET MAUI  in all places

File: MAUI/DigitalGauge/Character-Types.md [hunk 22]
```diff
@@ -0,0 +1,82 @@
...
+description: Learn here all about Character Types support in Syncfusion Maui Digital Gauge (SfDigitalGauge) control and more.
+platform: MAUI
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Character-Types.md:5]: platform: MAUI change as platform: maui **in all files**

File: MAUI/DigitalGauge/Character-Types.md [hunk 23]
```diff
@@ -0,0 +1,82 @@
...
+control: DigitalGauge
+documentation: ug
+---
+
+# Character Types in Maui Digital Gauge (SfDigitalGauge)
+
+The digital characters can be drawn in the following four different segments:
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Character-Types.md:12]: The digital characters can be display in the following four different segments:

File: MAUI/DigitalGauge/Character-Types.md [hunk 24]
```diff
@@ -0,0 +1,82 @@
...
+
+•	SevenSegment
+•	FourteenSegment
+•	SixteenSegment
+•	EightCrossEightDotMatrix
+
+## Seven segment
+
+The seven-segment type is capable of displaying numbers and a few uppercase letters efficiently.
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Character-Types.md:21]: What mean **few**?

File: MAUI/DigitalGauge/Character-Types.md [hunk 25]
```diff
@@ -0,0 +1,82 @@
...
+
+{% tabs %}
+
+{% highlight xaml %}
+
+     <gauge:SfDigitalGauge Text="1 2 3 4 5" StrokeWidth="5" 
+                          HeightRequest="100"  WidthRequest="300"
+                          DisabledSegmentAlpha="0.1" DisabledSegmentStroke="LightSkyBlue"
+                          CharacterHeight="90" CharacterWidth="25" 
+                          HorizontalOptions="Center" VerticalOptions="Center"
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Character-Types.md:31]: Add only  CharacterType="SevenSegment" API, alone

File: MAUI/DigitalGauge/Customize-character-segments.md [hunk 26]
```diff
@@ -0,0 +1,268 @@
...
+
+The values of digital characters are scaled by altering the height and width of digital characters. This is achieved by setting the `CharacterHeight` and `CharacterWidth` properties in the digital gauge. The default value of `CharacterHeight` is 60 and `CharacterWidth` is 40.
+`
+{% tabs %}
+
+{% highlight xaml %}
+
+    <gauge:SfDigitalGauge Text="SYNCFUSION" StrokeWidth="4" 
+                          HeightRequest="120"  WidthRequest="330" 
+                          BackgroundColor="Black"
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Customize-character-segments.md:24]: Only explain needed APIS's, mean DisabledSegmentAlpha is that need to add size customization

File: MAUI/DigitalGauge/Display-Types.md [hunk 27]
```diff
@@ -0,0 +1,165 @@
+---
+layout: post
+title: Display Types in Maui Digital Gauge control | Syncfusion
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Display-Types.md:3]: Display  Character Segment Types in .NET Maui Digital Gauge control

File: MAUI/DigitalGauge/Display-Types.md [hunk 28]
```diff
@@ -0,0 +1,165 @@
...
+description: Learn here all about Display Types support in Syncfusion Maui Digital Gauge (SfDigitalGauge) control and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+
+# Display Types in Maui Digital Gauge (SfDigitalGauge)
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Display-Types.md:10]: Character Display Types in .NET Maui Digital Gauge  (SfDigitalGauge)

File: MAUI/DigitalGauge/Display-Types.md [hunk 29]
```diff
@@ -0,0 +1,165 @@
...
+
+The digital gauge displays numbers, the alphabet, and special characters, which are given in the text property.
+
+## Setting value to number
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Display-Types.md:14]: Display number Character

File: MAUI/DigitalGauge/Display-Types.md [hunk 30]
```diff
@@ -0,0 +1,165 @@
...
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Setting value to alphabet
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Display-Types.md:66]: Display alphabets

File: MAUI/DigitalGauge/Display-Types.md [hunk 31]
```diff
@@ -0,0 +1,165 @@
...
+
+            digital.CharacterStroke = Color.Green;
+
+            digital.DisabledSegmentStroke = Color.Green;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Setting value to special characters
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Display-Types.md:116]: Display special characters

File: MAUI/DigitalGauge/Getting-Started.md [hunk 32]
```diff
@@ -0,0 +1,216 @@
...
+description: Learn here about getting started with Syncfusion Maui Digital Gauge (SfDigitalGauge) control, its elements and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
+---
+# Getting Started with Maui Digital Gauge (SfDigitalGauge)
+
+This section explains how to add the .NET MAUI DigitalGauge control. This section covers only the basic features needed to get started with Syncfusion DigitalGauge.
+
+## Creating an application using the .NET MAUI DigitalGauge
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Getting-Started.md:13]: NET MAUI Digital Gauge

File: MAUI/DigitalGauge/Character-Types.md [hunk 33]
```diff
@@ -0,0 +1,82 @@
...
+
+            digital.DisabledSegmentStroke = Color.LightSkyBlue;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Fourteen segment
+
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Character-Types.md:73]: Add example code

File: MAUI/DigitalGauge/Customize-character-segments.md [hunk 34]
```diff
@@ -0,0 +1,268 @@
...
+            digital.CharacterStroke = Color.FromRgb(20, 108, 237);
+
+            digital.DisabledSegmentStroke = Color.FromRgb(20, 108, 237);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Customize character segment stroke and stroke width
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Customize-character-segments.md:117]: 2 topics
Customize character segment stroke width
Customize character segment stroke

File: MAUI/DigitalGauge/Getting-Started.md [hunk 35]
```diff
@@ -0,0 +1,216 @@
...
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %} 
+
+## Setting character type for digital gauge
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Getting-Started.md:91]: Remove
 Setting character type for digital gauge
 ## Configuring properties

File: MAUI/DigitalGauge/Getting-Started.md [hunk 36]
```diff
@@ -0,0 +1,216 @@
...
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }
+    }
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/Getting-Started.md:86]: Add only Display Value in Digital Gauge

File: maui-toc.html [hunk 37]
```diff
@@ -590,6 +590,18 @@
...
             <li><a href="/maui/DateTime-Slider/events-and-commands">Events and Commands</a></li>
         </ul>
+    </li>
+	<li>
+        SfDigitalGauge
+        <ul>
+            <li><a href="/maui/Digital-Gauge/overview">Overview</a></li>
+            <li><a href="/maui/Digital-Gauge/getting-started">Getting Started</a></li>
+            <li><a href="/maui/Digital-Gauge/migration">Migration</a></li>
+            <li><a href="/maui/Digital-Gauge/customize-character-segments">Customize Character Segments</a></li>
```
Comment (Jeyasri-Murugan) [inline maui-toc.html:600]: Customize Character Segments to Character Segment Types

File: maui-toc.html [hunk 38]
```diff
@@ -590,6 +590,18 @@
...
+            <li><a href="/maui/Digital-Gauge/character-types">Character Types </a></li>
+            <li><a href="/maui/Digital-Gauge/digital-types">Digital Types </a></li>
+            <li><a href="/maui/Digital-Gauge/event">Event</a></li>
```
Comment (Jeyasri-Murugan) [inline maui-toc.html:603]: Event to Events

File: maui-toc.html [hunk 39]
```diff
@@ -590,6 +590,18 @@
...
         </ul>
+    </li>
+	<li>
+        SfDigitalGauge
+        <ul>
+            <li><a href="/maui/Digital-Gauge/overview">Overview</a></li>
+            <li><a href="/maui/Digital-Gauge/getting-started">Getting Started</a></li>
+            <li><a href="/maui/Digital-Gauge/migration">Migration</a></li>
+            <li><a href="/maui/Digital-Gauge/customize-character-segments">Customize Character Segments</a></li>
+            <li><a href="/maui/Digital-Gauge/character-types">Character Types </a></li>
```
Comment (Jeyasri-Murugan) [inline maui-toc.html:601]: Character Types to Character Segment Types

File: MAUI/DigitalGauge/overview.md [hunk 40]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: About Maui Digital Gauge control | Syncfusion
+description: Learn here all about introduction of Syncfusion Maui Digital Gauge (SfDigitalGauge) control, its elements and more.
+platform: MAUI
+control: DigitalGauge
+documentation: ug
```
Comment (Jeyasri-Murugan) [inline MAUI/DigitalGauge/overview.md:7]: Add keywords

**Pull request closed without merging**